dist/gaia-header.js

```diff
@@ -770,7 +771,7 @@ module.exports = component.register('gaia-header', {
       min: MINIMUM_FONT_SIZE_CENTERED
     });
 
-    var overflowing = fontFitResult.textWidth > textSpace;
+    var overflowing = fontFitResult.overflowing;
```

Comment:
uses the change in the font-fit lib

the issue was that we did not use the same bound in both places: in font-fit we were using a little buffer to account for canvas measurement issues, but not here, and as a result we still had ellipsis in some places. So this is fixing it.